### PR TITLE
Audit of trapped ions and sc qubits

### PR DESCRIPTION
**Pull request by alvaro-at-xanadu**
**Title:** Trapped ion quantum computing, Quantum computing with superconducting qubits

**Summary:** Updating these tutorials

**Relevant references:**

**Possible Drawbacks:**

**Related GitHub Issues:**

### CODE REVIEW

File: demonstrations/tutorial_sc_qubits.py [hunk 1]
```diff
@@ -4,13 +4,15 @@
...
+    :property="og:description": Learn about quantum computers based on superconducting qubits, developed by companies such as IBM and Google.
     :property="og:image": https://pennylane.ai/qml/_images/sc_qubits.png
 
 .. related::
+   tutorial_pasqal Quantum computation with neutral atoms
    tutorial_trapped_ions Quantum computation with trapped ions
+   tutorial_photonics Photonic quantum computers
 
-*Author: PennyLane dev team. Posted: 22 March 2022. Last updated: 22 March 2022.*
+*Author: Alvaro Ballon. Posted: 22 March 2022. Last updated: 22 March 2022.*
```
Comment (KetpuntoG) [inline demonstrations/tutorial_sc_qubits.py:15]:
```suggestion
*Author: Alvaro Ballon. Posted: 22 March 2022. Last updated: 26 August 2022.*
```

File: demonstrations/tutorial_trapped_ions.py [hunk 2]
```diff
@@ -4,18 +4,21 @@
...
+    :property="og:description": Learn all about trapped ion quantum computers, developed by companies such as IonQ and Honeywell.
     :property="og:image": https://pennylane.ai/qml/_images/trapped_ions_tn.png
 
 .. related::
    tutorial_pasqal Quantum computation with neutral atoms
+   tutorial_sc_qubits Quantum computing with superconducting qubits
+   tutorial_photonics Photonic quantum computers
 
-*Author: PennyLane dev team. Posted: 10 November 2021. Last updated: 10 November 2021.*
+*Author: Alvaro Ballon. Posted: 10 November 2021. Last updated: 30 August 2022.*
```
Comment (KetpuntoG) [inline demonstrations/tutorial_trapped_ions.py:15]: August 30 for some reason?

**Review by KetpuntoG — APPROVED** [review]
Looks good to me :) 
Just the date comment

File: demonstrations/tutorial_trapped_ions.py (hunk 2 above)
Comment (alvaro-at-xanadu) [inline demonstrations/tutorial_trapped_ions.py:15]:
```suggestion
*Author: Alvaro Ballon. Posted: 10 November 2021. Last updated: 26 August 2022.*
```

**Pull request merged**